actionpack/lib/action_dispatch/http/mime_negotiation.rb

```diff
@@ -151,6 +154,14 @@ def valid_accept_header
       def use_accept_header
         !self.class.ignore_accept_header
       end
+
+      private
+        def extract_variant!
+          if parameters.key?(:format)
+            parameters[:format], *variants = parameters[:format].to_s.split("+")
+            self.variant = variants.map(&:to_sym) unless variants.empty?
```

Comment:
Yes, logically. But `any?` will call itself with an implicit `{ |obj| obj }` block: http://ruby-doc.org/core-2.2.0/Enumerable.html#method-i-any-3F.

I doubt it's a problem here, given the tiny sizes variants could be.